src/services/SeedService.ts

```diff
@@ -241,8 +241,8 @@ export class SeedService {
       // convert from ISO string to Unix epoch seconds
       Date.parse(config.launchDetails.endDate) / 1000,
       [config.launchDetails.vestingPeriod, config.launchDetails.vestingCliff],
-      !!config.launchDetails.whitelist,
-      toWei(SeedService.seedFee),
+      config.launchDetails.isPermissoned,
+      toWei(config.launchDetails.seedTip ?? 0.0),
```

Comment:
How is `seedTip` stored in the contract? Is it BigNumber (uint256)? couldn't find any reference for that.
Should it be on a scale of 0 - 100 or 0.0 - 1.0?

Reply:
Let's go with what the tipping input spits out to us (eg. what `value` from the input returns)